modules/configuration/src/test/java/org/apache/ignite/internal/configuration/notifications/ConfigurationAnyListenerTest.java

```diff
@@ -656,4 +695,69 @@ void testAnyGetConfigFromNotificationEventOnUpdate() throws Exception {
 
         rootConfig.elements().get(key0).elements2().get(key1).intVal().update(newVal).get(1, SECONDS);
     }
+
+    @Test
+    void testAnyCreateNamedPolymorphicConfig() throws Exception {
+        AtomicBoolean invokeListener = new AtomicBoolean();
+
+        rootConfig.polyNamed().any().listen(configListener(ctx -> {
+            invokeListener.set(true);
```

Comment:
Shouldn't this be in the end?

Reply:
Not necessarily, it is important for me to know that we definitely hit the listener, if there is an exception below, then we will receive it while waiting for the configuration change.

Do you think it should be left at the end?

Reply:
Add tests:
- org.apache.ignite.internal.configuration.notifications.ConfigurationListenerTest#testGetErrorFromListener
- org.apache.ignite.internal.configuration.notifications.ConfigurationListenerTest#testGetErrorFromListenerFuture